src/server/routes/submit.route.js

```diff
@@ -13,37 +13,61 @@ const submitRouter = () => {
 
   router.get("", async (req, res) => {
     logEmitter.emit("functionCall", "Routes", "/submit route");
-    const controllerResponse = await submitController(
-      req.session.council,
-      req.session.cumulativeFullAnswers,
-      req.session.addressLookups,
-      req.session.pathConfig._id
-    );
-
-    req.session.submissionDate = controllerResponse.submissionDate;
-    req.session.fsaRegistrationNumber =
-      controllerResponse.fsaRegistrationNumber;
-    req.session.emailFbo = controllerResponse.emailFbo;
-    req.session.lcConfig = controllerResponse.lcConfig;
-
-    logEmitter.emit(
-      "functionSuccessWith",
-      "Routes",
-      "/submit route",
-      controllerResponse.redirectRoute
-    );
-    if (controllerResponse.redirectRoute === "back") {
-      res.redirect("back");
-    }
-    req.session.save(err => {
-      if (err) {
-        logEmitter.emit("functionFail", "Routes", "/submit route", err);
-        throw err;
-      }
-      res.redirect(
-        `/new/${req.session.council}${controllerResponse.redirectRoute}`
+
+    if (req.session.submissionSucceeded) {
```

Comment:
Becasue of the logging issue, could we factor this out into a function alreadySubmitted which return bool and inside can therefore log that this function has been called and, if not previously submitted, whether the sessions saving suceeded/failed? Test would be needed.